site/content/en/contributions/design/rate-limit.md

```diff
@@ -429,12 +429,12 @@ attribute such as [IP subnet][] in the future that are not relevant in the [HTTP
   * A Status field within the RateLimitFilter API will be added to reflect whether the specific configuration was programmed correctly in these multiple locations or not.
 
 [PolicyAttachment]: https://gateway-api.sigs.k8s.io/references/policy-attachment/
```

Comment:
Looks like `references` should be updated to `reference`. There's a redirect in place but that's what is causing issues.